### PR TITLE
Add Lotus models

### PR DESCRIPTION
**Pull request by mrewers**
Addresses Issue #6

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -3232,6 +3232,146 @@ const allSoloCars = {
...
+      'all': ['dp']
+    },
+    'Elan (RWD)': {
+      'all': ['cs', 'ssp']
+    },
+    'Elan (FWD)': {
+      'all': ['ssp']
+    },
+    'Elise': {
+      'all': ['ssm'],
```
Comment (Bjorn248) [inline common.js:3261]: Please remove the `all` and add `ssm` to every year instead.

File: common.js [hunk 2]
```diff
@@ -3232,6 +3232,146 @@ const allSoloCars = {
...
+    'Evora S': {
+      '2010': ['ssp'],
+      '2011': ['as', 'ssp', 'ssr'],
+      '2012': ['as', 'ssp', 'ssr'],
+      '2013': ['as', 'ssp', 'ssr'],
+      '2014': ['as', 'ssp', 'ssr'],
+      '2015': ['as', 'ssr']
+    },
+    'Evora': {
+      'all': ['ssm'],
```
Comment (Bjorn248) [inline common.js:3335]: Please remove the `all` and add `ssm` to every year.

File: common.js [hunk 3]
```diff
@@ -3232,6 +3232,146 @@ const allSoloCars = {
...
+      '2010': ['bs', 'ssp'],
+      '2011': ['bs', 'ssp'],
+      '2012': ['bs', 'ssp'],
+      '2013': ['bs', 'ssp'],
+      '2014': ['bs', 'ssp'],
+      '2015': ['bs']
+    },
+    'Exige': {
+      'all': ['ssm'],
```
Comment (Bjorn248) [inline common.js:3344]: Please remove the `all` and add `ssm` to every year.

File: common.js [hunk 4]
```diff
@@ -3232,6 +3232,146 @@ const allSoloCars = {
...
+    'Elite 2+2': {
+      'all': ['ssp']
+    },
+    'Elite (1216 cc)': {
+      'all': ['csp']
+    },
+    'Esprit': {
+      'all': ['cs', 'ssm']
+    },
+    'Esprit (4-cyl, V8)': {
```
Comment (Bjorn248) [inline common.js:3297]: Remove this model and add `ssp` to both the `Esprit Turbo` and the `Esprit` please

File: common.js [hunk 5]
```diff
@@ -3232,6 +3232,146 @@ const allSoloCars = {
...
+      '2000': ['as'],
+      '2001': ['as'],
+      '2002': ['as'],
+      '2003': ['as'],
+      '2004': ['as']
+    },
+    'Europa': {
+      'all': ['cs', 'ssp']
+    },
+    'Europa (Renault 1470 cc/1565 cc & Lotus-Ford Twin Cam 1558 cc)': {
```
Comment (Bjorn248) [inline common.js:3314]: Remove this and add `dp` to the `Europa` model, thanks!

File: common.js [hunk 6]
```diff
@@ -3232,6 +3232,146 @@ const allSoloCars = {
...
+    '7 & 7A (948, 997, & 1098 cc)': {
+      'all': ['dp']
+    },
+    'Cortina': {
+      'all': ['csp', 'hs']
+    },
+    'Eclat': {
+      'all': ['cs', 'ssp']
+    },
+    'Elan': {
```
Comment (Bjorn248) [inline common.js:3248]: Combine all Elan models into this one and remove the others. The classes should be `cs, ssp, dp`. Thanks!

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Just merging some models and years and this should be good to go!

**Comment by mrewers** [comment]
Resolved the issues and rebased off of master.

**Comment by Bjorn248** [comment]
Thanks!

**Pull request merged**